sky/serve/serve_utils.py

```diff
@@ -786,6 +819,14 @@ def get_service_status(cls, service_names: Optional[List[str]]) -> str:
         ]
         return cls._build(code)
 
+    @classmethod
+    def add_version(cls, service_name: str) -> str:
+        code = [
+            f'msg = serve_state.add_version({service_name!r})',
+            'print(msg, end="", flush=True)'
```

Comment:
Would be better to encode the msg to avoid unexpected output from ssh to pollute the message. Use `common_utils.encode_payload` and `common_utils.decode_payload`, or other manual encoding method like what we do in https://github.com/skypilot-org/skypilot/blob/f78a16e254faeb3667916d22d55b073339152d09/sky/backends/cloud_vm_ray_backend.py#L3349